cowprotocol/cowswap
# Service worker enabling and emergency.js in case of cache problems

# Summary
The service work was disabled previously due to caching problems.
At the PR I have enabled it using original uniswap files and added some modifications.
Also there is `emergency.js` script in case of service worker caching problems.

  # To Test

1. Deploy a new version
- [ ] Check that service worker was updated
- [ ] Check that index.html and .js files cached by SW
2. Deploy a one more version
- [ ] Check that service worker was updated
- [ ] Check that index.html and .js files cached by SW
- [ ] Check that changes from the latest version were applied
